include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,190 @@
+{
+    /* Vulkan 1.0 Core  */
+    bool robustBufferAccess = false;
+    //VkBool32    fullDrawIndexUint32;
+    bool imageCubeArray = false;
+    //VkBool32    independentBlend;
+    bool geometryShader    = false;
+    //VkBool32    tessellationShader;
+    //VkBool32    sampleRateShading;
+    //VkBool32    dualSrcBlend;
+    bool logicOp = false;
+    bool multiDrawIndirect = false;
+    //VkBool32    drawIndirectFirstInstance;
+    //VkBool32    depthClamp;
+    //VkBool32    depthBiasClamp;
+    //VkBool32    fillModeNonSolid;
+    //VkBool32    depthBounds;
+    //VkBool32    wideLines;
+    //VkBool32    largePoints;
+    //VkBool32    alphaToOne;
+    bool multiViewport = false;
+    bool samplerAnisotropy = false;
+    //VkBool32    textureCompressionETC2;
+    //VkBool32    textureCompressionASTC_LDR;
+    //VkBool32    textureCompressionBC;
+    //VkBool32    occlusionQueryPrecise;
+    //VkBool32    pipelineStatisticsQuery;
+    //VkBool32    vertexPipelineStoresAndAtomics;
+    //VkBool32    fragmentStoresAndAtomics;
+    //VkBool32    shaderTessellationAndGeometryPointSize;
+    //VkBool32    shaderImageGatherExtended;
+    //VkBool32    shaderStorageImageExtendedFormats;
+    //VkBool32    shaderStorageImageMultisample;
+    //VkBool32    shaderStorageImageReadWithoutFormat;
+    //VkBool32    shaderStorageImageWriteWithoutFormat;
+    //VkBool32    shaderUniformBufferArrayDynamicIndexing;
+    //VkBool32    shaderSampledImageArrayDynamicIndexing;
+    //VkBool32    shaderStorageBufferArrayDynamicIndexing;
+    //VkBool32    shaderStorageImageArrayDynamicIndexing;
+    //VkBool32    shaderClipDistance;
+    //VkBool32    shaderCullDistance;
+    bool vertexAttributeDouble = false; // shaderFloat64
+    //VkBool32    shaderInt64;
+    //VkBool32    shaderInt16;
+    //VkBool32    shaderResourceResidency;
+    //VkBool32    shaderResourceMinLod;
+    //VkBool32    sparseBinding;
+    //VkBool32    sparseResidencyBuffer;
+    //VkBool32    sparseResidencyImage2D;
+    //VkBool32    sparseResidencyImage3D;
+    //VkBool32    sparseResidency2Samples;
+    //VkBool32    sparseResidency4Samples;
+    //VkBool32    sparseResidency8Samples;
+    //VkBool32    sparseResidency16Samples;
+    //VkBool32    sparseResidencyAliased;
+    //VkBool32    variableMultisampleRate;
+    bool inheritedQueries = false;
+
+    /* Vulkan 1.1 Core */
+    //VkBool32           storageBuffer16BitAccess;
+    //VkBool32           uniformAndStorageBuffer16BitAccess;
+    //VkBool32           storagePushConstant16;
+    //VkBool32           storageInputOutput16;
+    //VkBool32           multiview;
+    //VkBool32           multiviewGeometryShader;
+    //VkBool32           multiviewTessellationShader;
+    //VkBool32           variablePointersStorageBuffer;
+    //VkBool32           variablePointers;
+    //VkBool32           protectedMemory;
+    //VkBool32           samplerYcbcrConversion;
+    //VkBool32           shaderDrawParameters;
```

Comment:
We want this: 
> shaderDrawParameters specifies whether the implementation supports the SPIR-V DrawParameters capability. When this feature is not enabled, shader modules must not declare the SPV_KHR_shader_draw_parameters extension or the DrawParameters capability.

Reply:
yes but only `shaderDrawPArameters`

in OpenGL its the presence of `ARB_shader_draw_parameters` extension that tells you it exists (or GL 4.6, but please verify)

under GLES this is always false, it does not have such extension

Reply:
ok I checked its core in GL 4.6

Reply:
Where did you go to make sure that `its core in GL 4.6` ?

Reply:
if you know the extension(s) that provide a feature, then you can check
https://www.khronos.org/opengl/wiki/History_of_OpenGL#OpenGL_4.6_.282017.29

Shame a similar document doesn't exist for GLES